src/LambdaEventParser.ts

```diff
@@ -3,7 +3,7 @@ import {
   IAWSLambdaProxyIntegrationRequest,
   isLambdaProxyRequest,
   Omit
-} from "common-types";
+} from "./";
```

Comment:
we talked bout changing this to `./index` this will work better in many transpilers